tools/static-assets/skel-svelte/package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "@babel/runtime": "^7.15.3",
     "meteor-node-stubs": "^1.1.0",
-    "svelte": "^3.38.2"
+    "svelte": "3.27.0"
```

Comment:
Since the Svelte skeleton uses `zodern:melte`, it can use the latest svelte version (it loads svelte from the app instead of having its own copy so the version used by the app isn't very important).

Reply:
The issue is with svelte-meteor-data: https://github.com/rdb/svelte-meteor-data/pull/13